### ISSUE TITLE
`cfsignal` should respect proxy environment variables

### ISSUE THREAD

**Issue by etungsten** [bottlerocket-os/bottlerocket#2302]
**What I'd like:**
`cfsignal` to respect proxy environment variables when making API requests out to CloudFormation.

Reference https://github.com/bottlerocket-os/bottlerocket/pull/2227/commits/538de7a5529aeba719cd56ca16d6da58d1a6858e for the changes necessary to do this.

**Comment by stmcginnis** [comment]
@etungsten is this still needed?

**Comment by etungsten** [comment]
> @etungsten is this still needed?

Yes it is. We need to implement a custom solution for this until https://github.com/awslabs/aws-sdk-rust/issues/169 is resolved.

**Comment by arnaldo2792** [comment]
This was implemented in https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/234 (thanks @jpculp  🎉 )
